### PR TITLE
Rename e2e-ai-operator-ztp-ipv4-sno-ocp-410 to e2e-ai-operator-ztp-converged and make it required

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml:55]: can we fine-tune on what cases we're running it?
because as-is it will run even on docs changes

Comment (osherdp) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml:55]: to clarify, how central is this flow? should it run more frequently than capi/regular-ztp?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by osherdp** [comment]
also until the relevant PR gets merged:
/hold

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml
Comment (eranco74) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml:55]: I think it should run the same as the basic ZTP job.
I'll adjust it

**Review by osherdp — APPROVED** [review]

**Pull request merged**
